lib/cc2650.js

```diff
@@ -346,4 +358,65 @@ CC2650SensorTag.prototype.convertSimpleKeyData = function(data, callback) {
   callback(left, right, reedRelay);
 };
 
+// Listener on 'AudioChange'
+CC2650SensorTag.prototype.onAudioChange = function(data) {
+  this.saveAudioData(data, function(audio) {
+    this.emit('AudioChange', audio);
+  }.bind(this));
+};
+
+// Append audio data to a file
+CC2650SensorTag.prototype.saveAudioData = function(data, callback) {
+  tagId = this.uuid; // Useful when several tags are connected
```

Comment:
Why is the `tagId` object needed in the library, isn't every SensorTag object self contained?

It's also a bit weird that the file name is hard coded. What's the current experience/steps for using this?